app/models/authenticator.rb

```diff
@@ -1,24 +1,14 @@
 class Authenticator < Struct.new(:auth_hash, :session)
```

Comment:
This class was there before I added the ApiAuthenticator and WebAuthenticator models, and it doesn't function in quite the same way. I wonder if we should rename it to alleviate some potential confusion, especially since all three models are in the same directory. Maybe `LoginUser`?